spring-rabbit/src/main/java/org/springframework/amqp/rabbit/annotation/RabbitListener.java

```diff
@@ -89,11 +90,13 @@
  * @since 1.4
  * @see EnableRabbit
  * @see RabbitListenerAnnotationBeanPostProcessor
+ * @see RabbitListeners
  */
 @Target({ ElementType.TYPE, ElementType.METHOD, ElementType.ANNOTATION_TYPE })
 @Retention(RetentionPolicy.RUNTIME)
 @MessageMapping
 @Documented
+@Repeatable(RabbitListeners.class)
```

Comment:
It doesn't compile on Java < 8, but it's ok because we always compile with Java 8 - even though the target level is 6.